docs/source/demos/text_input.ipynb

```diff
@@ -38,7 +38,7 @@
     "import pandas as pd\n",
     "\n",
     "input_data = urlopen('https://featurelabs-static.s3.amazonaws.com/spam_text_messages_modified.csv')\n",
-    "data = pd.read_csv(input_data)\n",
+    "data = pd.read_csv(input_data)[:750]\n",
```

Comment:
Nice. Dang how big was this dataset?

Reply:
Ah, 3k rows. Not too bad. Truncating seems good to me tho

Reply:
Yea 3k isn't too bad but the problem is that the text featurizer takes up a lot of the run time. Perhaps when we add component caching, we can go back to the full 3k rows!